src/chrome/render-asset.sh

```diff
@@ -5,20 +5,23 @@ RENDER_SVG="$(command -v rendersvg)" || true
 INKSCAPE="$(command -v inkscape)" || true
 OPTIPNG="$(command -v optipng)" || true
 
-if "$INKSCAPE" --help | grep -e "--export-filename" > /dev/null; then
-  EXPORT_FILE_OPTION="--export-filename"
-elif "$INKSCAPE" --help | grep -e "--export-file" > /dev/null; then
-  EXPORT_FILE_OPTION="--export-file"
-elif "$INKSCAPE" --help | grep -e "--export-png" > /dev/null; then
-  EXPORT_FILE_OPTION="--export-png"
+if [[ -n "${INKSCAPE}" ]]; then
+  if "$INKSCAPE" --help | grep -e "--export-filename" > /dev/null; then
+    EXPORT_FILE_OPTION="--export-filename"
+  elif "$INKSCAPE" --help | grep -e "--export-file" > /dev/null; then
+    EXPORT_FILE_OPTION="--export-file"
+  elif "$INKSCAPE" --help | grep -e "--export-png" > /dev/null; then
+    EXPORT_FILE_OPTION="--export-png"
+  fi
 fi
 
 i="$1"
 
 echo "Rendering '$i.png'"
+
 if [[ -n "${RENDER_SVG}" ]]; then
   "$RENDER_SVG" --dpi 96 "$i.svg" "$i.png"
-else
+elif [[ -n "${INKSCAPE}" ]]; then
```

Comment:
since you touched this part of code i have an extra question -- should the `else` case handled somehow?

for example:

```bash
else
    echo "ERROR: Either resvg or inkscape should be installed."
    exit 1
fi
```

Reply:
Yeah, I should have done that. Thanks for the suggestion!

Reply:
I forgot that the [`./render-assets.sh`](https://github.com/nana-4/materia-theme/blob/master/render-assets.sh#L4-L7) is already checking if resvg or inkscape is installed.

Now I'm thinking I shouldn't have touched this, because adding the `else` part makes the code a bit complex and ugly (for me). I'll revert it.

Reply:
my bad, i was looking only on the diff